src/main/scala/com/github/mlangc/slf4zio/api/package.scala

```diff
@@ -236,6 +318,9 @@ package object api {
   def makeLogger[T](clazz: Class[_]): UIO[Logger] =
     UIO(getLogger(clazz))
 
+  def makeMarker(name: String): UIO[Marker] =
```

Comment:
I would drop this method in favor of `getMarker` returning returning `UIO[Marker]`.

Reply:
Ok. Will fix